packages/api/tests/lambdas/test-process-s3-dead-letter-archive.js

```diff
@@ -135,6 +157,23 @@ test('processDeadLetterArchive only deletes dead letters that process successful
   t.deepEqual(output.processingFailedKeys, [failingMessageKey]);
 });
 
+test.serial('processDeadLetterArchive processes a SQS Message', async (t) => {
```

Comment:
Ah, is this the only unit test we need to prove that we fixed the bug that PODAAC saw?

Reply:
Yes, this should prove the failure case/is the breaking test.